src/pages/responsible/_banner.tsx

```diff
@@ -34,11 +34,11 @@ const Banner = ({ header, p1, button_text }) => {
 
 export const RoleBanner = () => (
     <Banner
-        header={<Localize translate_text="New to trading?" />}
+        header={<Localize translate_text="_t_New to trading?_t_" />}
```

Comment:
Shouldn't we use function instead of component here and below?

Reply:
As I tested, it's working fine because `Banner.propTypes` were assigned as below:

`Banner.propTypes = {
    button_text: PropTypes.object,
    header: PropTypes.object,
    p1: PropTypes.object,
}`